web/src/views/Deposit/Deposit.tsx

```diff
@@ -24,6 +24,7 @@ const Deposit = () => {
   const [success, setSuccess] = useState('');
   const [error, setError] = useState('');
   const [isLoading, setIsLoading] = useState(false);
+  const [, updateAccounts] = useAtom(accountsAtom);
```

Comment:
You can use `useSetAtom` is you're only going to write to state.

Reply:
I was gonna use that, but wanted to keep the same standards throughout used in ATM withdraw 
https://github.com/project-error/pefcl/blob/8564c354b8e418c4405dccf50b18c1e22ddec284/web/src/views/ATM/ATM.tsx#L60
I can change it if you want lmk